code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/service/impl/ServiceDiscoveryApiServiceImpl.java

```diff
@@ -322,7 +324,12 @@ private void populateTmpfs(Map<String, Object> cattleServiceData, Map<String, Ob
                     if (map.get(key) != null) {
                         option = map.get(key).toString();
                     }
-                    list.add(key + ":" + option);
+                    if (option.isEmpty()) {
+                        list.add(key);
+                    }
+                    else {
```

Comment:
Stylistically, if elses should look like:
```
if (condition) {
    // something
} else {
    // something else
}
```
meaning: put the else statement on the same line as the closing bracket for the if